tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -49,16 +49,16 @@
 #define PT_REGS_PARM6(x) (((PT_REGS_ARM64 *)(x))->regs[5])
 #endif
 
-#define MAX_PERCPU_BUFSIZE  (1 << 15)     // This value is actually set by the kernel as an upper bound
-#define MAX_STRING_SIZE     4096          // Choosing this value to be the same as PATH_MAX
-#define MAX_BYTES_ARR_SIZE  4096          // Max size of bytes array, arbitrarily chosen
-#define MAX_STR_ARR_ELEM    40            // String array elements number should be bounded due to instructions limit
-#define MAX_PATH_PREF_SIZE  64            // Max path prefix should be bounded due to instructions limit
-#define MAX_STACK_ADDRESSES 1024          // Max amount of different stack trace addresses to buffer in the Map
-#define MAX_STACK_DEPTH     20            // Max depth of each stack trace to track
-#define MAX_STR_FILTER_SIZE 16            // Max string filter size should be bounded to the size of the compared values (comm, uts)
-#define FILE_MAGIC_HDR_SIZE 16            // Number of bytes to save from a file's header (for magic_write event)
-#define FILE_MAGIC_MASK     15            // Mask used to pass verifier when submitting magic_write event bytes
+#define MAX_PERCPU_BUFSIZE      (1 << 15)     // This value is actually set by the kernel as an upper bound
+#define MAX_STRING_SIZE         4096          // Choosing this value to be the same as PATH_MAX
+#define MAX_BYTES_ARR_SIZE      4096          // Max size of bytes array, arbitrarily chosen
+#define MAX_STR_ARR_ELEM        40            // String array elements number should be bounded due to instructions limit
+#define MAX_PATH_PREF_SIZE      64            // Max path prefix should be bounded due to instructions limit
+#define MAX_STACK_ADDRESSES     1024          // Max amount of different stack trace addresses to buffer in the Map
+#define MAX_STACK_DEPTH         20            // Max depth of each stack trace to track
+#define MAX_STR_FILTER_SIZE     16            // Max string filter size should be bounded to the size of the compared values (comm, uts)
+#define FILE_MAGIC_HDR_SIZE     16            // Number of bytes to save from a file's header (for magic_write event)
+#define FILE_MAGIC_MASK         15            // Mask used to pass verifier when submitting magic_write event bytes
```

Comment:
Any reason to change this identation?

Reply:
No, i'm not actually sure how this happened? I'm reverting